<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowButtonDialog</class>
 <widget class="QDialog" name="MainWindowButtonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>341</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255)) }</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="btnOpen">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>8</pointsize>
            <italic>true</italic>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>Open Solution</string>
          </property>
          <property name="icon">
           <iconset resource="dbm.qrc">
            <normaloff>:/images/actions/images/big/bigfileopen.png</normaloff>:/images/actions/images/big/bigfileopen.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCommandLinkButton" name="btnQuick1">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>8</pointsize>
               <italic>false</italic>
               <kerning>false</kerning>
              </font>
             </property>
             <property name="text">
              <string>1</string>
             </property>
             <property name="icon">
              <iconset resource="dbm.qrc">
               <normaloff>:/images/actions/images/big/bigfileopen.png</normaloff>:/images/actions/images/big/bigfileopen.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCommandLinkButton" name="btnQuick2">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>8</pointsize>
               <italic>false</italic>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>2</string>
             </property>
             <property name="icon">
              <iconset resource="dbm.qrc">
               <normaloff>:/images/actions/images/big/bigfileopen.png</normaloff>:/images/actions/images/big/bigfileopen.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCommandLinkButton" name="btnQuick3">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>8</pointsize>
               <italic>false</italic>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>3</string>
             </property>
             <property name="icon">
              <iconset resource="dbm.qrc">
               <normaloff>:/images/actions/images/big/bigfileopen.png</normaloff>:/images/actions/images/big/bigfileopen.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCommandLinkButton" name="btnQuick4">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>8</pointsize>
               <italic>false</italic>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>4</string>
             </property>
             <property name="icon">
              <iconset resource="dbm.qrc">
               <normaloff>:/images/actions/images/big/bigfileopen.png</normaloff>:/images/actions/images/big/bigfileopen.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCommandLinkButton" name="btnQuick5">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>8</pointsize>
               <italic>false</italic>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>5</string>
             </property>
             <property name="icon">
              <iconset resource="dbm.qrc">
               <normaloff>:/images/actions/images/big/bigfileopen.png</normaloff>:/images/actions/images/big/bigfileopen.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="btnNew">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>8</pointsize>
            <italic>true</italic>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>New Solution</string>
          </property>
          <property name="icon">
           <iconset resource="dbm.qrc">
            <normaloff>:/images/actions/images/big/bigfilenew.png</normaloff>:/images/actions/images/big/bigfilenew.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="btnNewConnection">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>8</pointsize>
            <italic>true</italic>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>New Connection</string>
          </property>
          <property name="icon">
           <iconset resource="dbm.qrc">
            <normaloff>:/images/actions/images/actions/new_dbconn.png</normaloff>:/images/actions/images/actions/new_dbconn.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="btnHelp">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>8</pointsize>
            <italic>true</italic>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>Getting started</string>
          </property>
          <property name="icon">
           <iconset resource="dbm.qrc">
            <normaloff>:/images/actions/images/small/help.png</normaloff>:/images/actions/images/small/help.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="dbm.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>MainWindowButtonDialog</sender>
   <signal>rejected()</signal>
   <receiver>MainWindowButtonDialog</receiver>
   <slot>onCloseWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOpen</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowButtonDialog</receiver>
   <slot>onOpenProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNew</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowButtonDialog</receiver>
   <slot>onNewProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNewConnection</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowButtonDialog</receiver>
   <slot>onNewConnection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnHelp</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowButtonDialog</receiver>
   <slot>onHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onNewProject()</slot>
  <slot>onCloseWindow()</slot>
  <slot>onOpenProject()</slot>
  <slot>onNewConnection()</slot>
  <slot>onHelp()</slot>
 </slots>
</ui>
